docs/api/components/RouterLink.md

```diff
@@ -7,7 +7,7 @@ RouterLink component renders anchor tag (`<a>`) for routing both within the SPA
 | Parameter | Type |
 | :---- | :---- |
 | to | [`Url`](/api/types/Url) \| `(resolve: RouterResolve) => Url` |
-| query | `Record<string, string>` \| `undefined` |
+| query | `ConstructorParameters<typeof URLSearchParams>[0]` \| `undefined` |
```

Comment:
We should be using the `QueryRecord` (which I'm suggesting we rename to `QuerySource` everywhere and not this constructor type.